examples/controlnet/train_controlnet_mlflow.py

```diff
@@ -553,9 +553,15 @@ def parse_args(input_args=None):
     parser.add_argument(
         "--logging_steps",
         type=int,
-        default=10,
+        default=100,
         help="Log every X updates steps. Default to 10.",
     )
+    parser.add_argument(
+        "--mlflow_uri",
+        type=str,
+        default='http://127.0.0.1:5001',
+        help="URI to log mlflow.",
+    )
```

Comment:
I think a CLI argument for mlflow tracking uri is not necessary. Just use the env var in shell script & testbench

Reply:
Yeah, I will remove it in next PR